### PR TITLE
Updating CurationConcerns to 0.12.0.pre1, fixing deprecation warnings

### LINKED ISSUE #109: Update Curation Concerns

### PR DESCRIPTION
**Pull request by escowles**
Closes #109

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.2.1
```
Comment (tpendragon) [inline .ruby-version]: This necessary?

Comment (escowles) [inline .ruby-version]: rbenv wants it -- I can put it in .gitignore.

But we're already specifying 2.2.1 in https://github.com/projecthydra-labs/geo_concerns/pull/111/files/bd011649f94ff3537f9b36c6bb478bedc3e6f229#diff-8b7db4d5cc4b8f6dc8feb7030baa2478L2

So if `Gemfile` demands 2.2.1, then `.ruby-version` needs to match.  Maybe remove them both?

Comment (tpendragon) [inline .ruby-version]: I like the idea of removing them both.

Comment (escowles) [inline .ruby-version]: Sounds good — better not to tie to a particular version — updated PR on the way.

**Comment by escowles** [comment]
I've updated the PR to remove the 2.2.1 requirement (also expanded a controller test to cover a new line that wasn't tested).

File: .ruby-version (hunk 1 above)
Comment (eliotjordan) [inline .ruby-version]: Yep. No reason to specify 2.2.1.

**Pull request merged**